src/ResourceManager/Profile/Commands.Profile.Test/DefaultCmdletTests.ps1

```diff
@@ -0,0 +1,107 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests Set-AzureRmDefault when resource group given does not exist
+#>
+function Test-SetAzureRmDefaultResourceGroupNonexistent
```

Comment:
I completely redid the test - I figured I would test all of resource group in one test because all the commands are interconnected so you can't really test one alone effectively.